osl/interface_linux.go

```diff
@@ -30,6 +31,86 @@ type nwIface struct {
 	sync.Mutex
 }
 
+func (i *nwIface) MarshalJSON() ([]byte, error) {
```

Comment:
@mavenugo  I agree that we should restore running containers with as less states to store as possible and to reconstruct states from running instances, this design has tried to follow this rule,
at this design we doesn't store any state of the driver, all the driver's state are reconstruct from the running instances. 
But for some states, we can't reconstruct from the the running instances